plothist/plothist_style.py

```diff
@@ -201,3 +201,72 @@ def get_fitting_ylabel_fontsize(ax):
         ax.get_yaxis().get_label().set_size(ylabel_fontsize)
 
     return ylabel_fontsize
+
+
+def add_text(
+    text,
+    x="left",
+    y="top",
+    fontsize=12,
+    white_background=False,
+    ax=None,
+    **kwargs,
+):
+    """
+    Add text to an axis.
+
+    Parameters
+    ----------
+    text : str
+        The text to add.
+    x : float, optional
+        x position, can be a float in [0, 1] or "left"/"right" to put the text to the left (x=0) or right (x=1) of the plot, by default "left".
+    y : float, optional
+        y position, can be a float in [0, 1] or "top"/"bottom" to put the text to the top (y=1.01) or bottom (y=0) of the plot, by default "top".
```

Comment:
It is not necessarily in [0,1]. What about

y position in unit of the y-axis length. The default is value "top", which is an alias for 1.01. The other alias "bottom" corresponds to 0.0.